x-pack/plugin/async-search/src/main/java/org/elasticsearch/xpack/search/MutableSearchResponse.java

```diff
@@ -84,13 +91,16 @@ class MutableSearchResponse {
     /**
      * Updates the response with the result of a partial reduction.
      * @param reducedAggs is a strategy for producing the reduced aggs
+     * @param isFinalLocalReduce true if the local cluster search has finished (during CCS, this can be true, but the overall
+     *                           search is still running on remote clusters)
```

Comment:
add "in case roundtrips are minimized" ?

Reply:
Good catch. Fixed.